### DISCUSSION TITLE
QMenu style and class not considered

### DISCUSSION THREAD

**Discussion by zentoac** [quasarframework/quasar#10426] [General - Components / Directives / etc]
Hello,
I can't make this piece of code work. `style` and `class` attributes are not considered.
I've already put this code in a single component without anything else.

"quasar": "^1.15.23",
"@quasar/extras": "^1.10.9",
"node": ">= 10.18.1",
"npm": ">= 6.13.4",
"yarn": ">= 1.21.1"

```
<template>
    <q-btn color="primary" label="Open Menu">
      <q-menu class="bg-red" :style="{ backgroundColor: '#eee', color: 'blue'}">
        <q-list style="min-width: 100px">
          <q-item clickable v-close-popup>
            <q-item clickable>
              <q-item-section>New tab</q-item-section>
            </q-item>
          </q-item>
        </q-list>
      </q-menu>
    </q-btn>
</template>
```
Any idea?
Thanks

**Comment by pdanpdan** [comment]
Please check the API card here: https://v1.quasar.dev/vue-components/menu#qmenu-api

You even have examples.

**Reply by zentoac** [reply]
I copied the code from that!
by the way it seems that the `:content-style="{ backgroundColor: '#eee', color: 'blue'}"` attribute works but its not documented in the QMenu API.

**Reply by zentoac** [reply]
Anyway I'm trying to update the max-width of a sub menu, so that it overlaps the parent menu (I don't like the `cover` attribute effect and also the `max-width` attribute doesn't work) but it doesn't work with `:content-style="{ maxWidth: '300px'}"`
How can I do it runtime when the QMenu opens? Are there other ways?
Thanks
